src/main/resources/db/postgres/V1_19__aktiviteter.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE AKTIVITETER
```

Comment:
There is already 19 in dev, so maybe we can remove that one since its not necessary, and cleanup firefly tables...